Cargo.toml

```
@@ -13,7 +13,7 @@ keywords = ["C","expression","parser"]
 travis-ci = { repository = "jethrogb/rust-cexpr" }
 
 [dependencies]
-nom = { version = "6", default-features = false, features = ["std"] }
+nom = { version = "7.0", default-features = false, features = ["std"] }
```

Since `nom` follows semantic versioning, `7` should work (which would also allow users to use the non-breaking `7.1` once released):

```suggestion
nom = { version = "7", default-features = false, features = ["std"] }
```

---

These two specifications are equivalent.